src/v0/destinations/bqstream/transform.js

```diff
@@ -21,7 +21,7 @@ const getInsertIdColValue = (properties, insertIdCol) => {
   return null;
 };
 
-const process = event => {
+const process = async (event) => {
```

Comment:
@ItsSudip async is not really required. This was removed as part of the last review comment.

Reply:
Removed the async can you check?